components/data/librarysize/index.tsx

```diff
@@ -7,10 +7,10 @@ import { PlotlyPlot } from '@/components/viz/plotly'
 
 
 
-export const librarysize = MetaNode('librarysize')
+export const VisualizeLibrarySize = MetaNode('VisualizeLibrarySize')
   .meta({
-    label: 'librarysize',
-    description: 'librarysize',
+    label: 'VisualizeLibrarySize',
+    description: 'VisualizeLibrarySize',
```

Comment:
Since the `label` and `description` are shown to the user, they should have spaces between words and make it clear to the user what they do. These will appear on the interface